config/crd/bases/kmm.sigs.k8s.io_modules.yaml

```diff
@@ -3,7 +3,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     api-approved.kubernetes.io: unapproved, testing-only
-    controller-gen.kubebuilder.io/version: v0.9.2
+    controller-gen.kubebuilder.io/version: v0.8.0
```

Comment:
Please upgrade to a newer `controller-gen` and run `make manifests`.

Reply:
fixed